--- a/erts/configure.ac
+++ b/erts/configure.ac
@@ -23,7 +23,7 @@ dnl autoconf convention that should be cleaned up.
 
 AC_INIT
 AC_CONFIG_SRCDIR([emulator/beam/erl_process.c])
-AC_PREREQ([2.71])
+AC_PREREQ([2.69])
 
 m4_include([otp.m4])
 
diff --git a/lib/common_test/configure.ac b/lib/common_test/configure.ac
index 490a64328d..460778860b 100644
--- a/lib/common_test/configure.ac
+++ b/lib/common_test/configure.ac
@@ -1,5 +1,5 @@
 AC_INIT
-AC_PREREQ([2.71])
+AC_PREREQ([2.69])
 
 m4_include([otp.m4])
 
diff --git a/lib/common_test/test_server/configure.ac b/lib/common_test/test_server/configure.ac
index da5c1b1608..c900dc9938 100644
--- a/lib/common_test/test_server/configure.ac
+++ b/lib/common_test/test_server/configure.ac
@@ -21,7 +21,7 @@ dnl
 
 AC_INIT
 AC_CONFIG_SRCDIR([conf_vars.in])
-AC_PREREQ([2.71])
+AC_PREREQ([2.69])
 
 AC_CANONICAL_HOST
 
diff --git a/lib/crypto/configure.ac b/lib/crypto/configure.ac
index 52d1f6c324..650c78f876 100644
--- a/lib/crypto/configure.ac
+++ b/lib/crypto/configure.ac
@@ -24,7 +24,7 @@ dnl define([AC_CACHE_SAVE], )dnl
 
 AC_INIT
 AC_CONFIG_SRCDIR([vsn.mk])
-AC_PREREQ([2.71])
+AC_PREREQ([2.69])
 
 m4_include([otp.m4])
 
diff --git a/lib/erl_interface/configure.ac b/lib/erl_interface/configure.ac
index d24e3f48eb..c16f548450 100644
--- a/lib/erl_interface/configure.ac
+++ b/lib/erl_interface/configure.ac
@@ -27,7 +27,7 @@ dnl m4_define(EI_VERSION,regexp(m4_include(VERSION),[version \([-.0-9A-Za-z]+\)]
 
 AC_INIT
 AC_CONFIG_SRCDIR([include/ei.h])
-AC_PREREQ([2.71])
+AC_PREREQ([2.69])
 
 m4_include([otp.m4])
 
diff --git a/lib/megaco/configure.ac b/lib/megaco/configure.ac
index ce65e54a85..b5f57efc4b 100644
--- a/lib/megaco/configure.ac
+++ b/lib/megaco/configure.ac
@@ -25,7 +25,7 @@ dnl define([AC_CACHE_SAVE], )dnl
 
 AC_INIT
 AC_CONFIG_SRCDIR([vsn.mk])
-AC_PREREQ([2.71])
+AC_PREREQ([2.69])
 
 m4_include([otp.m4])
 
diff --git a/lib/odbc/configure.ac b/lib/odbc/configure.ac
index dfc3a6ec04..0635455d62 100644
--- a/lib/odbc/configure.ac
+++ b/lib/odbc/configure.ac
@@ -25,7 +25,7 @@ dnl define([AC_CACHE_SAVE], )dnl
 dnl Process this file with autoconf to produce a configure script.
 AC_INIT
 AC_CONFIG_SRCDIR([c_src/odbcserver.c])
-AC_PREREQ([2.71])
+AC_PREREQ([2.69])
 
 m4_include([otp.m4])
 
diff --git a/lib/snmp/configure.ac b/lib/snmp/configure.ac
index c87c38dc68..c8971f3e92 100644
--- a/lib/snmp/configure.ac
+++ b/lib/snmp/configure.ac
@@ -4,7 +4,7 @@ define([AC_CACHE_SAVE], )dnl
 
 AC_INIT
 AC_CONFIG_SRCDIR([src/app/snmp.app.src])
-AC_PREREQ([2.71])
+AC_PREREQ([2.69])
 
 m4_include([otp.m4])
 
diff --git a/lib/wx/configure.ac b/lib/wx/configure.ac
index d49399c540..0909cee206 100644
--- a/lib/wx/configure.ac
+++ b/lib/wx/configure.ac
@@ -24,7 +24,7 @@ m4_include([otp.m4])
 
 AC_CONFIG_AUX_DIR([${ERL_TOP}/make/autoconf])
 
-AC_PREREQ([2.71])
+AC_PREREQ([2.69])
 
 ## Delete previous failed configure results
 if test -f ./CONF_INFO; then
diff --git a/make/configure.ac b/make/configure.ac
index e0d4103b6e..a4130a427a 100644
--- a/make/configure.ac
+++ b/make/configure.ac
@@ -18,7 +18,7 @@ dnl limitations under the License.
 dnl
 dnl %CopyrightEnd%
 
-AC_PREREQ([2.71])dnl
+AC_PREREQ([2.69])dnl
 
 AC_INIT
 
diff --git a/otp_build b/otp_build
index 7b35b39fc2..7fed8449ed 100755
--- a/otp_build
+++ b/otp_build
@@ -19,7 +19,7 @@
 # %CopyrightEnd%
 #
 
-USE_AUTOCONF_VERSION=2.71
+USE_AUTOCONF_VERSION=2.69
 
 autoconf_aux_dirs="./erts/autoconf ./lib/common_test/test_server"
 
